api/app/src/main/kotlin/packit/controllers/PacketGroupController.kt

```diff
@@ -4,15 +4,17 @@ import org.springframework.data.domain.Page
 import org.springframework.http.ResponseEntity
 import org.springframework.stereotype.Controller
 import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestParam
 import packit.model.PageablePayload
 import packit.model.dto.PacketGroupDto
+import packit.model.dto.PacketIdAndDisplayName
 import packit.model.toDto
 import packit.service.PacketGroupService
 
 @Controller
-@RequestMapping("/packetGroup")
+@RequestMapping("/packetGroups")
```

Comment:
by convention I think resources are plural in CRUD; in any case we already use the plural form for the packets controller.